src/counts.jl

```diff
@@ -9,7 +9,16 @@
 typealias IntUnitRange{T<:Integer} UnitRange{T}
 
 #### functions for counting a single list of integers (1D)
+"""
+    addcounts!(r::AbstractArray, x::IntegerArray, levels::IntUnitRange)
 
+### Args:
+* `r`: Accumulating array.
+* `x`: An `IntegerArray` for which the counts are computed.
+* `levels`: The range of values to check for occurence in `x`. 
+
+This function adds the counts of values in `levels` that appear in `x` to an accumulating array `r` or a map.
```

Comment:
Do you mean "to an accumulating array or a map `r`"? I think you should either be more explicit about what a map is, or remain vague by mentioning only "array".